### DISCUSSION TITLE
Landscape Support

### DISCUSSION THREAD

**Discussion by skitzofrenzy** [dailydotdev/daily#1209] [Feature Requests]
I've noticed the this app is only capable of displaying in portrait which is fine in most cases, but recently I've started using my tab)(less distraction) and that usually lives in landscape and I sometimes have Google read the articles out loud so the tab just suits that better when I'm working

**Comment by idoshamun** [comment]
Hi @skitzofrenzy,

I assume you use the mobile app right? Which device exactly?

**Reply by skitzofrenzy** [reply]
Hey, I have a tab s8+
